libraries/testkit/cluster.py

```diff
@@ -405,3 +437,42 @@ def validate_cluster(sync_gateways, sg_accels, config):
         return False, "INVALID CONFIG: Running in Distributed Index mode but no sg_accels are defined."
 
     return True, ""
+
+
+def get_buckets_from_sync_gateway_config(sync_gateway_config_path):
```

Comment:
just wondering, we already have function to get buckets from sync gateway config in config.py. Isn't not possible to use that method?

Reply:
it will create loop.

Reply:
I'm changing the import to use it from syncgateway.py

Reply:
looks like this method is still not imported from sync gateway